backend/src/procedure/mod.rs

```diff
@@ -64,6 +64,15 @@ impl PackageProcedure {
         allow_inject: bool,
         timeout: Option<Duration>,
     ) -> Result<Result<O, (i32, String)>, Error> {
+        tracing::trace!(
+            "Procedure execute {} {} - {:?}",
+            match self {
+                PackageProcedure::Docker(_) => "docker",
+                PackageProcedure::Script(_) => "JS",
+            },
+            pkg_id,
+            name
+        );
```

Comment:
yeesh this is a lot of visual space for a trace log. Is there any way to make it more innocuous?
I may encourage display instances for `Package Procedure` and then you can eliminate the case statement and probably smash this to a single line.

Reply:
Thanks for the idea. It did reduce the vertical garbage